### ISSUE TITLE
fix(ally): change border color for weak badge/tag

### ISSUE THREAD

**Issue by mfranzke** [db-ux-design-system/core-web#3887]
### Which generators are impacted?

- [x] All
- [ ] HTML
- [ ] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

- open https://db-ux-design-system.github.io/core-web/version/v1.1.0/components/feedback/badge/overview?page=content
- you'll recognize that the dot is hard to identify due to a lack of color contrast

### Expected Behaviour

The border color for badge weak is compliant to WCAG minimum contrast ratio of 3:1

### Screenshots

<img width="98" alt="Image" src="https://github.com/user-attachments/assets/7b2af669-1081-482c-b34f-e492f870a42e" />

### Browser version

None

### Add any other context about the problem here.

DBUXDS-4400
